composer.json

```diff
@@ -1,66 +1,81 @@
 {
-    "name": "doctrine/migrations",
-    "type": "library",
-    "description": "PHP Doctrine Migrations project offer additional functionality on top of the database abstraction layer (DBAL) for versioning your database schema and easily deploying changes to it. It is a very easy to use and a powerful tool.",
-    "keywords": ["php", "database", "migrations", "dbal"],
-    "homepage": "https://www.doctrine-project.org/projects/migrations.html",
-    "license": "MIT",
-    "authors": [
-        {"name": "Benjamin Eberlei", "email": "kontakt@beberlei.de"},
-        {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
-        {"name": "Michael Simonson", "email": "contact@mikesimonson.com" }
-    ],
-    "require": {
-        "php": "^7.2",
-        "doctrine/dbal": "^2.10",
-        "doctrine/event-manager": "^1.0",
-        "composer/package-versions-deprecated": "^1.8",
-        "ocramius/proxy-manager": "^2.0.2",
-        "psr/log": "^1.1.3",
-        "symfony/console": "^3.4||^4.4.16||^5.0",
-        "symfony/stopwatch": "^3.4||^4.0||^5.0"
+  "name": "doctrine/migrations",
```

Comment:
Let's revert to 4 spaces to make the diff minimal here? `composer normalize` is not supposed to complain as far as I understood, because it detects it:

> format the normalized content (either as sniffed, or as specified using the --indent-size and --indent-style options)

Reply:
4 spaces was the reason I reformatted the .json file in the first place.  Note that I added composer-normalize configs into composer.json 

```json
    "composer-normalize": {
      "indent-size": 2,
      "indent-style": "space"
    }
```

Reply:
But why? I think we use 4 spaces throughout Doctrine packages, and that's also
[what Composer documents](https://getcomposer.org/doc/01-basic-usage.md), and
what I've seen in the ecosystem, but maybe you have a different experience? 
I think the usage of composer-normalize could be generalized, and that we could
do that bikeshedding in a more centralized way to ensure consistency. I'd
recommend you stick with 4 spaces here, and then open a PR against
https://github.com/doctrine/.github with 2 spaces (and arguments for that),
that way we can discuss this once and do that change in a consistent way rather
than just on this package.

Reply:
compser.json is now 4 spaces